v4-client-py/v4_client_py/clients/helpers/chain_helpers.py

```diff
@@ -25,6 +26,41 @@ class OrderExecution(Flag):
     POST_ONLY = auto()  # Post-only
     FOK = auto()        # Fill or Kill
 
+# Enums to use in order proto fields. Use proto generated fields once that's fixed.
+class Order_TimeInForce(Flag):
+
+    '''
+    TIME_IN_FORCE_UNSPECIFIED - TIME_IN_FORCE_UNSPECIFIED represents the default behavior where an
+    order will first match with existing orders on the book, and any
+    remaining size will be added to the book as a maker order.
+    '''
+    TIME_IN_FORCE_UNSPECIFIED = 0
+
+    '''
+    TIME_IN_FORCE_IOC - TIME_IN_FORCE_IOC enforces that an order only be matched with
+    maker orders on the book. If the order has remaining size after
+    matching with existing orders on the book, the remaining size
+    is not placed on the book.
+    '''
+    TIME_IN_FORCE_IOC = 1
+
+    '''
+    TIME_IN_FORCE_POST_ONLY - TIME_IN_FORCE_POST_ONLY enforces that an order only be placed
+    on the book as a maker order. Note this means that validators will cancel
+    any newly-placed post only orders that would cross with other maker
+    orders.
+    '''
+    TIME_IN_FORCE_POST_ONLY = 2
+
+    '''
+    TIME_IN_FORCE_FILL_OR_KILL - TIME_IN_FORCE_FILL_OR_KILL enforces that an order will either be filled
+    completely and immediately by maker orders on the book or canceled if the
+    entire amount can‘t be matched.
+    '''
+    TIME_IN_FORCE_FILL_OR_KILL = 3
+
+    UNRECOGNIZED = -1
```

Comment:
Suggestion: any reason why this shouldn't be removed? I'm not sure if this actually protects against anything